inmaputil/download.go

```diff
@@ -0,0 +1,105 @@
+package inmaputil
+
+import (
+	"github.com/mholt/archiver"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/cavaliercoder/grab"
+)
+
+func Download(path string) string {
+	// Check if local file exists
```

Comment:
All comments within functions should be full sentences with capitals at the beginning and periods at the end.